### DISCUSSION TITLE
Sonoff Pow R2 Multiple switch toggeling after restart

### DISCUSSION THREAD

**Discussion by jin-gonzu** [arendst/Tasmota#17760] [Q&A]
Hello, in the last weeks I had trouble with my Sonoff Pow R2, which i used for a few years with Tasmota now.

It began by switching off suddenly. I updated the firmware, I reset my MQTT broker. I reset the Tasmota Sonoff device.
But so far nothing helped, looking at the console logs, it shows that the device is switching between states fast.

13:20:18.462 MQT: stat/sonoff_TV/RESULT = {"POWER":"ON"}
13:20:19.087 MQT: tele/sonoff_TV/STATE (..)
13:20:19.093 MQT: stat/sonoff_TV/RESULT = {"POWER":"OFF"}

Which i find very confusing.
I used an mqtt programm to look at the data send between the device and the broker. 

There are no new commands send to the device.

Currently with retaining messages disabled, and all retain messages removed I get this result above. 
The state should be "ON" and it is for a brief moment before it is "OFF" again.

Any idea is appreciated.
I can reproduce the behaviour by typing restart 1 in my console. 

EDIT:
RESOLVED

**Comment by Jason2866** [comment]
To help we need a full log.

**Reply by jin-gonzu** [reply]
Thanks, the issue is resolved, I will try to find out how to close it

**Reply by Jason2866** [reply]
This is github discussions, Q&A can not be closed. Providing info what the problem was and mark as answer
